### DISCUSSION TITLE
[enh] improve the show page for services

### DISCUSSION THREAD

**Discussion by madoleary** [tosdr/edit.tosdr.org#1047] [Ideas]
The information, boxes and buttons at the top of the page could be organized more cleanly.

**Comment by madoleary** [comment]
I just finished a first draft of this. The wikipedia links on the live website don't go anywhere, FYI. I didn't include previous versions, as the way versions are currently displayed isn't very readable, and I'm working on that in the reason branch. I also didn't include the tabs for pending points, as there appears to be a bug with the javascript as mentioned in issue #274 and i'd like to work on that in a different branch.

**Comment by madoleary** [comment]
if this layout is agreeable, i'd like to implement it in other areas of the application. i think it's a bit more extendable then our current tables, which aren't actually tables and are just made up of many smaller "card" components.

**Comment by evilcel3ri** [comment]
> The wikipedia links on the live website don't go anywhere

This depends if there is a wikipedia linked given by the user. The wikipedia link is very new, only the latest service have it. 

> I also didn't include the tabs for pending points, as there appears to be a bug with the javascript as mentioned in issue #274 and i'd like to work on that in a different branch.

This a small feature that could be improved. I am not confortable with javascript, so if you find a way of doing it, it'd be great :D 

Also, what do you mean you didn't include the tabs for pending points ? The idea behind those are a to have a small way to have easily the pending points first for the curator.

**Comment by madoleary** [comment]
Then does it make sense to only display the Wikipedia link if it exists in the database?

I mean the tabs functionality isn't on the service show page in this specific commit since there seems to be an issue with how it interacts with the "order by rating" button. Will try to get that figured out this week.

**Comment by evilcel3ri** [comment]
> Then does it make sense to only display the Wikipedia link if it exists in the database?

Yep sure ! It was a quick hotfix after the bootcamp, it could be integrated better

> I mean the tabs functionality isn't on the service show page in this specific commit since there seems to be an issue with how it interacts with the "order by rating" button.

Maybe there could be a way to refactor the order button and call it on when necessary ?

**Comment by evilcel3ri** [comment]
Is it still relevant ?

**Comment by Vinnl** [comment]
@madoleary Can you use any help with this? Is that first draft somewhere public?
